lunatrace/bsl/frontend/src/dependency-tree/builds-dependency-tree.spec.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright by LunaSec (owned by Refinery Labs, Inc)
+ *
+ * Licensed under the Business Source License v1.1
+ * (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ * https://github.com/lunasec-io/lunasec/blob/master/licenses/BSL-LunaTrace.txt
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import { DependencyTree } from './builds-dependency-tree';
+import { BuildDependencyPartial } from './types';
+
+it('should generate a dependency tree', () => {
+  const dependencies: Array<BuildDependencyPartial> = [
```

Comment:
this would typically be in a fixtures file with jest, like how the backend is. Ideally that would be a folder within the test folder that jest is configured to ignore and then this would be a `.ts` file in there.  That way you could make it way bigger too, if you want. Not that it needs to be, but it shouldn't be right in the `it`.  I like the effort here to manually create a fake tree with some known outputs, thats a nice fixture

Also, this `it` should be wrapped in a describe. "The dependency tree" "should generate"

Reply:
Mostly done. I put a TODO for the fixtures.